### ISSUE TITLE
Pretrained model on Visual Genome dataset

### ISSUE THREAD

**Issue by shipra25jain** [nexusapoorvacus/DeepVariationStructuredRL#6]
Hi,
Is it possible to get checkpoints of this model for visual genome dataset?

Thank you

**Comment by kyteinsky** [comment]
Yeah that would be of great help @nexusapoorvacus @shipra25jain, we wouldn't be wasting resources for the very same objective.
